Tests/AppTests/SearchFilterTests.swift

```diff
@@ -126,10 +126,10 @@ class SearchFilterTests: AppTestCase {
         let filter = try StarsSearchFilter(value: "1", comparison: .greaterThan)
         let builder = SQLSelectBuilder(on: app.db as! SQLDatabase)
             .where(searchFilters: [filter])
-        let sql = renderSQL(builder, resolveBinds: true)
-        _assertInlineSnapshot(matching: sql, as: .lines, with: """
-        SELECT  WHERE ("stars" > '1')
-        """)
+        _assertInlineSnapshot(matching: renderSQL(builder), as: .lines, with: """
+            SELECT  WHERE ("stars" > $1)
+            """)
+        XCTAssertEqual(binds(builder), ["1"])
```

Comment:
These tests are somewhat redundant now that I added `test_packageMatchQuery_StarsSearchFilter` et al to `SearchTests`. Keeping them in for now but I have some follow-up changes for another day that'll address this.